.github/workflows/build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         run: |
           go test -race -count=1 -coverprofile=queue.coverprofile ./queue
           go test -race -count=1 -coverprofile=server.coverprofile  ./server
-          go test -race -count=1 -coverprofile=main.coverprofile
+          go test -race -count=1 -coverprofile=main.coverprofile -short
```

Comment:
wait a sec, why short?

Reply:
Because new test eats too much memory
https://github.com/allegro/bigcache/pull/236/files#diff-013414c814f8b4b2df17143a678bcc6726758e92f0be44d01df40e3dca9ad967R1072